toxcore/tox.h

```diff
@@ -4384,11 +4389,14 @@ typedef enum Tox_Err_Group_Send_Message {
  * @param message A non-NULL pointer to the first element of a byte array
  *   containing the message text.
  * @param length Length of the message to be sent.
+ * @param pseudo_msg_id A pointer to a uint32_t. the pseudo_msg_id of this message will be returned
+ *   unless the paramter is NULL. then the returned paramter value will be undefined.
```

Comment:
Odd grammar. "...unless the parameter is NULL, in which case the..."

Reply:
done